### PR TITLE
Closes #2235 - PyTest Benchmark for stream

### LINKED ISSUE #2235: PyTest Benchmark for Stream
Configure pytest benchmark for bigint_stream.py and stream.py

### PR DESCRIPTION
**Pull request by joshmarshall1**
This PR closes #2235 

- Adds benchmark `stream_benchmark.py`
- Combines `bigint_stream.py` and `stream.py` benchmarks into a single benchmark

### CODE REVIEW

**Review by Ethan-DeBandi99 — CHANGES_REQUESTED** [review]
It does not look like the `--randomize` option was pulled over here. We should maintain that in the event that you want a random set, but don't set the `seed`.

**Comment by joshmarshall1** [comment]
> It does not look like the `--randomize` option was pulled over here. We should maintain that in the event that you want a random set, but don't set the `seed`.

Added in the --randomize flag

**Review by Ethan-DeBandi99 — APPROVED** [review]

File: tests/conftest.py [hunk 1]
```diff
@@ -39,6 +39,14 @@ def pytest_addoption(parser):
         help="Benchmark only option. Only applies to bigint testing."
              "Maximum number of bits, so values > 2**max_bits will wraparound. -1 is interpreted as no maximum."
     )
+    parser.addoption(
+        "--alpha", action="store", default="1",
```
Comment (pierce314159) [inline tests/conftest.py:43]: in the original alpha is a real value, i just wanna make sure we'll still be multiplying by `1.0` here

Comment (joshmarshall1) [reply tests/conftest.py:43]: Added the `1.0` multiplication to the non-bigint cases - verified with @pierce314159 that bigint can't accept a `real` multiplication so it has to remain `int` in that case

**Review by pierce314159 — APPROVED** [review]

**Review by jaketrookman — APPROVED** [review]

**Pull request merged**